lib/service/desktop.go

```diff
@@ -222,10 +222,11 @@ func (process *TeleportProcess) initWindowsDesktopServiceRegistered(logger *slog
 		Labels:       cfg.WindowsDesktop.Labels,
 		HostLabelsFn: cfg.WindowsDesktop.HostLabels.LabelsForHost,
 		Heartbeat: desktop.HeartbeatConfig{
-			HostUUID:    cfg.HostUUID,
-			PublicAddr:  publicAddr,
-			StaticHosts: cfg.WindowsDesktop.StaticHosts,
-			OnHeartbeat: process.OnHeartbeat(teleport.ComponentWindowsDesktop),
+			HostUUID:        cfg.HostUUID,
+			PublicAddr:      publicAddr,
+			StaticHosts:     cfg.WindowsDesktop.StaticHosts,
+			OnHeartbeat:     process.OnHeartbeat(teleport.ComponentWindowsDesktop),
```

Comment:
I'm not sure if this is necessary or if this is only meant to be used when heartbeating for a service.